subsys/net/ip/tcp.c

```diff
@@ -28,11 +28,23 @@ LOG_MODULE_REGISTER(net_tcp, CONFIG_NET_TCP_LOG_LEVEL);
 
 #define ACK_TIMEOUT_MS CONFIG_NET_TCP_ACK_TIMEOUT
 #define ACK_TIMEOUT K_MSEC(ACK_TIMEOUT_MS)
+
+#ifndef CONFIG_NET_TCP_RANDOMIZED_RTO
```

Comment:
Cound you swap the ifndef to ifdef, at least for me it is easier to read the normal check instead of the nagated one.

Reply:
Done

Reply:
Did you forget to push? This also doesn't seem to be resolved.

Reply:
Dito